README.md

```diff
@@ -19,7 +19,7 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details.
 
 ## Sub-projects Documentation
  - oneAPI Data Parallel C++ compiler - See
-   [GetStartedGuide.md](sycl/doc/GetStartedGuide.md)
+   - [DPC++ Documentation](https://intel.github.io/llvm-docs/)
```

Comment:
I suggest to have a single line here:
 - oneAPI Data Parallel C++ compiler - See [DPC++ Documentation](https://intel.github.io/llvm-docs/)